test/test_version.py

```diff
@@ -0,0 +1,67 @@
+import logging
```

Comment:
```suggestion
# Copyright (C) 2022 Intel Corporation
# SPDX-License-Identifier: GPL-3.0-or-later

import logging
```

This should probably have a copyright/licensing header.  The copyright owner can be you rather than Intel if you want, since you're writing it.